includes/track1.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Track  1 Layout</title>
+    <style>
+        .container {
+            width: 800px;
+        }
+
+        svg {
+            border: 1px solid black;
+        }
+
+        rect {
+            fill:#000000;
+        }
+
+        .path {
+            fill:#ffffff;
+        }
+
+        rect.finish {
+            fill: #ff1c1c;
+        }
+
+        button {
+            position: relative;
+            top: 780px;
+            left: 480px;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <button>Return</button>
+        <svg width="1000" height="800">
+            <!--Background-->
+            <rect x="0" y="0" width="1000" height="800" class="bg" />
```

Comment:
class bg does not exist in styles.
Also, if it is referring to background, change the name to be more semantic, bg is not very clear.

Reply:
good point, well made.